docs/docs/integrations/embedding-stores/pgvector.md

```diff
@@ -17,12 +17,63 @@ https://github.com/pgvector/pgvector
 </dependency>
 ```
 
+## Gradle Dependency
+
+```implementation 'dev.langchain4j:langchain4j-pgvector:0.36.2'```
 
 ## APIs
 
 - `PgVectorEmbeddingStore`
 
+## Parameter Summary
+
+| Plain Java Property      | Spring Application Property | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Default Value       | Required/Optional                                                                                                                                                                                                                                                                                |
+|--------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `datasource`             | `coming soon`               | The `DataSource` object used for database connections.                                                                                                                                                                                                                                                                                                                                                                                                         | None                | Required if `host`, `port`, `user`, `password`, and `database` are not provided individually.                                                                                                                                                                                                    |

```

Comment:
ordering of properties could be improved: first required, then optional. First most important, then less important. `datasource` should be close to `host`/`port`/etc. `table` and `createTale` should be closee to each other, etc

Reply:
done